src/fileformat_pb.jl

```diff
@@ -0,0 +1,104 @@
+# Automatically generated by ProtoBuf.jl
+# syntax: proto2
+using ProtoBuf
+import ProtoBuf.meta
+
+mutable struct Blob <: ProtoType
+    __protobuf_jl_internal_meta::ProtoMeta
+    __protobuf_jl_internal_values::Dict{Symbol,Any}
```

Comment:
I do not know whether this is just controlled by `ProtoBuf`, but if we control it could we, for performance reasons, use some concrete type or a type `Union` perhaps?

Reply:
This is generated by ProtoBuf.jl, I was also surprised that it would not create concrete type fields instead of a `Dict`, I thought the whole idea behind ProtoBuf was to generate concrete specialized structs with fields. I suggest we stay with this with this PR and then try to change it to concrete fields and see what's the performance speedup in another PR. This separate PR could be a nice story to report to ProtoBuf.jl if we get a significant speedup